### PR TITLE
consensys.net -> consensys.io

### PR DESCRIPTION
**Pull request by kevin-chassagne**
## **Description**

Update consensys.net url to consensys.io

## **Related issues**

https://github.com/Consensys/dx-team/issues/1703

## **Manual testing steps**

n/a

## **Screenshots/Recordings**

n/a

### **Before**

n/a

### **After**

n/a

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by kevin-chassagne** [comment]
I have read the CLA Document and I hereby sign the CLA

File: shared/lib/ui-utils.js [hunk 1]
```diff
@@ -4,13 +4,12 @@ export const SUPPORT_LINK = process.env.SUPPORT_LINK;
...
-export const PRIVACY_POLICY_LINK = 'https://consensys.net/privacy-policy/';
+export const PRIVACY_POLICY_LINK = 'https://consensys.io/privacy-policy/';
 
 // TODO make sure these links are correct
 export const ETHERSCAN_PRIVACY_LINK = 'https://etherscan.io/privacyPolicy';
-export const CONSENSYS_PRIVACY_LINK = 'https://consensys.net/privacy-policy/';
-export const AUTO_DETECT_TOKEN_LEARN_MORE_LINK =
-  'https://consensys.net/privacy-policy/';
+export const CONSENSYS_PRIVACY_LINK = 'https://consensys.io/privacy-policy/';
+export const AUTO_DETECT_TOKEN_LEARN_MORE_LINK = 'https://consensys.io/privacy-policy/';
```
Comment (vthomas13) [inline shared/lib/ui-utils.js:12]:
```suggestion
export const AUTO_DETECT_TOKEN_LEARN_MORE_LINK = 
  'https://consensys.io/privacy-policy/';
```

Comment (vthomas13) [reply shared/lib/ui-utils.js:12]: This suggestion is due to the lint step failing in the pipeline

**Review by vthomas13 — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Comment by Anouk1973** [comment]
I k snap er niks van wil mn geld en account sluiten

Op do 4 jan. 2024 23:46 schreef Danica Shen ***@***.***>:

**Pull request merged**